README.md

```diff
@@ -2,14 +2,15 @@
 
 [![npm version](https://img.shields.io/npm/v/storybook-addon-preview.svg?style=flat-square&color=007acc&label=Version)](https://badge.fury.io/js/storybook-addon-preview)
 
-Storybook Addon Preview can show user selected [knobs](https://github.com/storybookjs/storybook/tree/master/addons/knobs) in various framework code in [Storybook](https://storybook.js.org/)
+Storybook Addon Preview can show user selected [args(controls)](https://github.com/storybookjs/storybook/tree/next/addons/controls) or [knobs](https://github.com/storybookjs/storybook/tree/master/addons/knobs) in various framework code in [Storybook](https://storybook.js.org/)
```

Comment:
Why don't you use the master branch's document?
I checked it and it is identical to the next branch's
https://github.com/storybookjs/storybook/tree/master/addons/controls